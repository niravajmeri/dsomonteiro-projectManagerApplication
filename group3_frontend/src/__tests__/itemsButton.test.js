import React from 'react';
import { shallow } from 'enzyme';
import ItemsButton from '../components/projectsTable/itemsButton';

describe('ItemsButton', () => {
    it('should render correctly', () => {
        const component = shallow(
            <ItemsButton href="ref" text="component text" />
        );
        expect(component).toMatchSnapshot();
    });
});
