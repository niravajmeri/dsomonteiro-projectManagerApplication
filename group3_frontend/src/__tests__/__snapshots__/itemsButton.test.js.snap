// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemsButton should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ItemsButton
    href="ref"
    text="component text"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "component text",
      "className": "itemsButton",
      "href": "ref",
      "type": "button",
    },
    "ref": null,
    "rendered": "component text",
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "component text",
        "className": "itemsButton",
        "href": "ref",
        "type": "button",
      },
      "ref": null,
      "rendered": "component text",
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
